sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -191,21 +191,52 @@ shouldBe('typeof gl.getProgramParameter(standardProgram, gl.VALIDATE_STATUS)', '
 shouldBe('gl.getProgramParameter(standardProgram, gl.ATTACHED_SHADERS)', '2');
 shouldBe('gl.getProgramParameter(standardProgram, gl.ACTIVE_ATTRIBUTES)', '2');
 shouldBe('gl.getProgramParameter(standardProgram, gl.ACTIVE_UNIFORMS)', '1');
+
+var validArrayForProgramParameter = [
+    gl.DELETE_STATUS,
+    gl.LINK_STATUS,
+    gl.VALIDATE_STATUS,
+    gl.ATTACHED_SHADERS,
+    gl.ACTIVE_ATTRIBUTES,
+    gl.ACTIVE_UNIFORMS
+]
 testInvalidArgument(
     "getProgramParameter",
     "parameter",
-    [ gl.DELETE_STATUS,
-      gl.LINK_STATUS,
-      gl.VALIDATE_STATUS,
-      gl.ATTACHED_SHADERS,
-      gl.ACTIVE_ATTRIBUTES,
-      gl.ACTIVE_UNIFORMS
-    ],
+    validArrayForProgramParameter,
     function(parameter) {
       return gl.getProgramParameter(standardProgram, parameter);
     }
 );
 
+if (contextVersion > 1) {
+  var buffer = gl.createBuffer();
+  gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, buffer);
+  gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 1024, gl.DYNAMIC_DRAW);
+  var transformFeedbackVars = ["normal", "ecPosition"];
+  gl.transformFeedbackVaryings(uniformBlockProgram, transformFeedbackVars, gl.INTERLEAVED_ATTRIBS);
+
+  var uniformBlockProgram = wtu.loadUniformBlockProgram(gl);
+  shouldBe('gl.getProgramParameter(standardProgram, gl.ACTIVE_UNIFORM_BLOCKS)', '1');
+
+  shouldBe('gl.getProgramParameter(standardProgram, gl.TRANSFORM_FEEDBACK_VARYINGS)', '2');
+  shouldBe('gl.getProgramParameter(standardProgram, gl.TRANSFORM_FEEDBACK_BUFFER_MODE)', 'gl.INTERLEAVED_ATTRIBS');
+
+  validArrayForProgramParameter += [
+      gl.ACTIVE_UNIFORM_BLOCKS,
+      gl.TRANSFORM_FEEDBACK_VARYINGS,
+      gl.TRANSFORM_FEEDBACK_BUFFER_MODE
+  ]
```

Comment:
These need to be added to the valid array before the first call to testInvalidArgument for getProgramParameter above, or that test will fail. Correct? It should not be necessary to duplicate that test in this if-block. Did you run this test on a WebGL 2 prototype and make sure it passed?